### PR TITLE
Fix cpu usage statistics

### PR DESCRIPTION
**Pull request by Megamouse**
- Fixes updating cpu usage in performance overlay during low detail mode
- Fixes division by zero in cpu_stats::get_usage (maybe the logic is wrong anyway)

### CODE REVIEW

File: rpcs3/util/cpu_stats.cpp [hunk 1]
```diff
@@ -80,8 +80,16 @@ namespace utils
...
 		double percent = 1. * (sys.QuadPart - m_sys_cpu) + (usr.QuadPart - m_usr_cpu);
-		percent /= (now.QuadPart - m_last_cpu);
-		percent /= utils::get_thread_count();
+
+		if (const double cpu = (now.QuadPart - m_last_cpu))
+		{
+			percent /= cpu;
+		}
+
+		if (const u32 thread_count = utils::get_thread_count())
```
Comment (elad335) [inline rpcs3/util/cpu_stats.cpp:89]: This is never zero, atleast shoud be never be zero.

Comment (Megamouse) [reply rpcs3/util/cpu_stats.cpp:89]: Who knows

Comment (Megamouse) [reply rpcs3/util/cpu_stats.cpp:89]: I can improve the code. One sec

Comment (Nekotekina) [reply rpcs3/util/cpu_stats.cpp:89]: I think non-windows code with division may have similar problem, although I never seen it.

Comment (Megamouse) [reply rpcs3/util/cpu_stats.cpp]: The non-windows code is already checking this.
I adjusted both codes paths to do the same.
The function now returns 0 in both cases.

**Pull request merged**